src/style.css

```
@@ -0,0 +1,38 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul {
+
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  list-style: none;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Combine into one rule
```suggestion
  background-image: url(./images/star-active.svg);
  background-repeat: no-repeat;
  background-position: center;
```